e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -179,6 +179,33 @@ describe('Upload Files', () => {
             cy.wait(TIMEOUTS.HALF_SEC);
         });
     });
+
+    it('MM-T337 CTRL/CMD+U - Five files on one message, thumbnails while uploading', () => {
```

Comment:
Failed on me running the test. Looks like need to login as sysadmin at the start of `beforeEach`.
<img width="721" alt="Screen Shot 2020-09-02 at 2 33 50 AM" src="https://user-images.githubusercontent.com/5334504/91892380-21eb4280-ecc5-11ea-91e5-c402f5f34578.png">

Reply:
i've updated the test,hopefully it'll be less flaky now

Reply:
I'm still getting a consistent error related to permission since it requires admin at the start of `beforeEach`.
<img width="703" alt="Screen Shot 2020-09-02 at 3 50 13 PM" src="https://user-images.githubusercontent.com/5334504/91955421-b8efe300-ed35-11ea-86c2-ac206637bf47.png">